lib/features/users/user_profile_screen.dart

```diff
@@ -0,0 +1,341 @@
+import 'package:faker/faker.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+import 'package:go_router/go_router.dart';
+import 'package:tiktok/constants/sizes.dart';
+import 'package:tiktok/features/users/widgets/persistent_tab_bar.dart';
+import 'package:tiktok/features/users/widgets/user_post_only_text.dart';
+import 'package:tiktok/features/users/widgets/user_post_with_image.dart';
+import 'package:tiktok/utils.dart';
+
+class UserProfileScreen extends ConsumerStatefulWidget {
+  const UserProfileScreen({
+    super.key,
+    required this.username,
+  });
+
+  final String username;
+
+  @override
+  UserProfileScreenState createState() => UserProfileScreenState();
+}
+
+class UserProfileScreenState extends ConsumerState<UserProfileScreen> {
+  final faker = Faker();
+  int selectedTab = 0;
+
+  void _onTabTapped(int index) {
+    setState(() {
+      selectedTab = index;
+    });
+  }
+
+  void _onGearPressed() {
+    context.push("/settings");
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        child: DefaultTabController(
+          length: 2,
+          child: NestedScrollView(
+            headerSliverBuilder: (context, innerBoxIsScrolled) {
+              return [
+                SliverAppBar(
+                  leading: IconButton(
+                    icon: FaIcon(
+                      FontAwesomeIcons.globe,
+                      color: isDarkMode(context, ref)
+                          ? Colors.white.withOpacity(0.8)
+                          : Colors.black.withOpacity(0.8),
+                    ),
+                    onPressed: null,
+                  ),
+                  actions: [
+                    IconButton(
+                      icon: const FaIcon(
+                        FontAwesomeIcons.instagram,
+                      ),
+                      onPressed: () {},
+                    ),
+                    IconButton(
+                      icon: const FaIcon(
+                        FontAwesomeIcons.gripLines,
+                      ),
+                      onPressed: () {
+                        _onGearPressed();
+                      },
+                    ),
+                  ],
+                ),
+                SliverToBoxAdapter(
+                  child: Column(
+                    children: [
+                      Row(
+                        children: [
+                          Padding(
+                            padding: const EdgeInsets.symmetric(
+                              horizontal: Sizes.size16,
```

Comment:
overflow error 발생합니다.

```suggestion
                              horizontal: Sizes.size14,
```

Reply:
해결했습니다~!